andela/wger-sparta
# 151593391: Bg exercise cache when deleting muscles

**What does this PR do?**
fix updating of cache after muscle deletion. 

**Description of task to be completed**
Make sure the muscles overview view updates immediately after the administrator 
deletes a muscle from the muscles view. 

**How should this be manually tested?**
    * Login to wger as an administrator. 
    * open the muscles view under the exercises tab and delete one of the muscles. 
    * click the muscle overview page under the exercises 
    * The deleted muscle won't appear in the muscle overview page. 

**What are the relevant pivotal tracker stories?**
#151593391
